elixir/apps/domain/lib/domain/mailer/sync_email/sync_error.html.eex

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:v="urn:schemas-microsoft-com:vml">
+	<head>
+		<meta charset="utf-8">
+		<meta name="x-apple-disable-message-reformatting">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="format-detection" content="telephone=no, date=no, address=no, email=no, url=no">
+		<meta name="color-scheme" content="light dark">
+		<meta name="supported-color-schemes" content="light dark">
+		<!--[if mso]>
+		<noscript>
+			<xml>
+				<o:OfficeDocumentSettings xmlns:o="urn:schemas-microsoft-com:office:office">
+					<o:PixelsPerInch>96</o:PixelsPerInch>
+				</o:OfficeDocumentSettings>
+			</xml>
+		</noscript>
+		<style>
+			td,th,div,p,a,h1,h2,h3,h4,h5,h6 {font-family: "Segoe UI", sans-serif; mso-line-height-rule: exactly;}
+		</style>
+		<![endif]-->
+		<title>Firezone Sync Error</title>
+		<style>
+			.hover-important-text-decoration-underline:hover {
+				text-decoration: underline !important
+			}
+			:is([dir="rtl"] .rtl-text-right) {
+				text-align: right !important
+			}
+			@media (max-width: 600px) {
+				.sm-my-8 {
+					margin-top: 32px !important;
+					margin-bottom: 32px !important
+				}
+				.sm-px-4 {
+					padding-left: 16px !important;
+					padding-right: 16px !important
+				}
+				.sm-px-6 {
+					padding-left: 24px !important;
+					padding-right: 24px !important
+				}
+				.sm-leading-8 {
+					line-height: 32px !important
+				}
+			}
+		</style>
+	</head>
+	<body style="margin: 0; width: 100%; background-color: #f6f6f6; padding: 0; -webkit-font-smoothing: antialiased; word-break: break-word">
+		<div role="article" aria-roledescription="email" aria-label="Firezone Sync Error" lang="en">
+			<div class="sm-px-4" style="background-color: #f6f6f6; font-family: ui-sans-serif, system-ui, -apple-system, 'Segoe UI', sans-serif">
+				<table align="center" cellpadding="0" cellspacing="0" role="none">
+					<tr>
+						<td style="width: 552px; max-width: 100%">
+							<div class="sm-my-8" style="margin-top: 48px; margin-bottom: 48px; text-align: center">
+								<a href="https://firezone.dev">
+									<div>
+										<img src="https://www.firezone.dev/images/logo-lockup.png" width="250" alt="Firezone logo" style="max-width: 100%; vertical-align: middle; line-height: 1">
+									</div>
+								</a>
+							</div>
+							<table style="width: 100%;" cellpadding="0" cellspacing="0" role="none">
+								<tr>
+									<td class="sm-px-6" style="border-radius: 4px; background-color: #ffffff; padding: 48px; font-size: 16px; color: #4f4f4f; box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.05)">
+										<h1 class="sm-leading-8" style="margin: 0 0 24px; font-size: 24px; font-weight: 600; color: #000000">
+											Identity Provider Sync Error!
```

Comment:
Might be clearer to have the IdP name in the body title and subject of the email

```suggestion
											<%= @provider.nane %> sync error!
```

Reply:
updated